samples/drivers/lora_send/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.13.1)
+macro(set_conf_file)
```

Comment:
I think this is no longer needed, the build system will pickup board specific .conf files automatically.

Reply:
Okay